src/customComponents/ElementListItem.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
```

Comment:
@CRIS7HIAN30-Femsa check this component, it could be used to render all user history records